misc/profiles2/lookups.dat

```diff
@@ -30,34 +30,30 @@ highway;0000039003 platform
 highway;0000037192 proposed planned virtual
 highway;0000010307 raceway
 highway;0000003152 rest_area
-highway;0000002942 abandoned disused razed demolished dismantled
+highway;0000002942 abandoned disused razed demolished dismantled no
 highway;0000002631 services
 highway;0000002133 corridor
-highway;0000002093 crossing
```

Comment:
It would be very unfortunate if this was removed. `highway=crossing` is used [more than 7 million times](https://taginfo.openstreetmap.org/tags/highway=crossing) and is useful in car, bike, and pedestrian profiles.

Reply:
Thanks for the review but please note that this is `highway=crossing` in way context, so, see the taginfo link that is only tagged 360 times right now and it looks to me like a tagging error, according to the [https://wiki.openstreetmap.org/wiki/Tag%3Ahighway%3Dcrossing](url) is it not allowed.

Reply:
You are right, sorry!

Reply:
My mistake! Sorry for the unnecessary objection.